tests/conftest.py

```diff
@@ -243,6 +249,22 @@ def random_filters(test_filters_random_with_mask: Filters) -> Filters:
     return test_filters_random_with_mask
 
 
+@pytest.fixture
+def synthetic_scar_removal(synthetic_scars_image: np.ndarray) -> Scars:
+    """Scars object using a synthetic image"""
+    scars = Scars(
+        img=synthetic_scars_image,
+        filename="",
+        run=True,
+        removal_iterations=2,
+        threshold_low=1.5,
+        threshold_high=1.8,
+        max_scar_width=2,
+        min_scar_length=1,
```

Comment:
For the `max_scar_width` and `min_scar_length`, yes. I think it's easier to debug and see what is going on if the script is allowed to count scars that are only 1 pixel wide, and also only two pixels tall. Otherwise there would be a lot more combinations of pixel patterns that I would need to make (because of the number of permutations increasing). 

For the other two variables, these are just what work as a discriminator between the low value pixels and the high value pixels that I placed in the image. 

All of this is to say that I created the image and the tests before trying it on real data, so once I found an algorithm that worked on the tests, I had to update it for the real data. I think that having differing values in the tests here is okay since the base functionality does not change when using different values.

Reply:
As I've commented on the #387 I think this is the correct way of doing things and we need to stop continually changing values in `topostats/default_config.yaml` whenever a new feature or refactor is undertaken.

If tests require specific values change those in the dictionary for the specific test but leave `default_config.yaml` as is. To which end I think #302 could probably be reopened for discussion as there hasn't been much and certainly not a consensus on what should be used (but that is likely the major problem with experimental conditions introducing variation that precludes having a one-size-fits-all configuration).